compose_test.go

```diff
@@ -68,7 +68,7 @@ func TestComposeGenerationWithoutDps(t *testing.T) {
 		GenerateComposeYaml([]string{})
 	})
 	should := []string{`(?m)^\s*periscope:`, `(?m)^\s*cluster-proxy:`, `(?m)^\s*datalake:`, `(?m)^\s*redbeams:`}
-	shouldnt := []string{`(?m)^\s*cloudbreak:`, `(?m)^\s*core-gateway:`}
+	shouldnt := []string{`(?m)^\s*cloudbreak:`, `(?m)^\s*core-gateway:`, `(?m)^\s*datalake-dr:`}
```

Comment:
**suggestion:** Regex tend to be inscrutable, it's often useful to provide an explanatory comment:
```
# match from the beginning of a string, an arbitrary number of whitespaces, then a service name followed by a litera `:`.